### ISSUE TITLE
System.TypeLoadException: Method 'AndThenTo' does not have an implementation.

### ISSUE THREAD

**Issue by tjaskula** [vlingo-net/vlingo-net-actors#34]
This happens with the following line `World.StartWithDefault("test-request-response-channel");`

```
System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.TypeLoadException: Method 'AndThenTo' in type 'Vlingo.Common.ResultCompletes`1' from assembly 'Vlingo.Actors, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null' does not have an implementation.
   at Vlingo.Actors.Actor..ctor()
   at Vlingo.Actors.PrivateRootActor..ctor()
   --- End of inner exception stack trace ---
   at System.RuntimeTypeHandle.CreateInstance(RuntimeType type, Boolean publicOnly, Boolean wrapExceptions, Boolean& canBeCached, RuntimeMethodHandleInternal& ctor)
   at System.RuntimeType.CreateInstanceSlow(Boolean publicOnly, Boolean wrapExceptions, Boolean skipCheckThis, Boolean fillCache)
   at System.RuntimeType.CreateInstanceDefaultCtor(Boolean publicOnly, Boolean skipCheckThis, Boolean fillCache, Boolean wrapExceptions)
   at System.Activator.CreateInstance(Type type, Boolean nonPublic, Boolean wrapExceptions)
   at System.Activator.CreateInstance(Type type, Boolean nonPublic)
   at System.Activator.CreateInstance(Type type)
   at Vlingo.Actors.ActorFactory.ActorFor(Stage stage, Actor parent, Definition definition, IAddress address, IMailbox mailbox, ISupervisor supervisor, ILogger logger)
   at Vlingo.Actors.Stage.CreateRawActor(Definition definition, Actor parent, IAddress maybeAddress, IMailbox maybeMailbox, ISupervisor maybeSupervisor, ILogger logger)
```

**Comment by tjaskula** [comment]
This works from UT in `Actors` but not in `Wire` any idea @zpbappi ?

**Comment by tjaskula** [comment]
Ok, I think this is because of version inconsistency on `Vlingo.Common` between `Vlingo.Actors` and `Vlingo.Wire` that is addressed in PR #35

**Comment by zpbappi** [comment]
@tjaskula yes, you are correct.

**Comment by tjaskula** [comment]
Resolved by #35
